client/my-sites/comments/comment/comment-author.jsx

```diff
@@ -29,6 +29,17 @@ export class CommentAuthor extends Component {
 		isExpanded: PropTypes.bool,
 	};
 
+	commentHasLink = () => {
+		if ( typeof DOMParser !== 'undefined' && DOMParser.prototype.parseFromString ) {
```

Comment:
Looks like there's decent support for DOMParser 👍

Reply:
this is brilliant! You learn something every day :)